bazel/repositories_extra.bzl

```diff
@@ -69,7 +69,7 @@ def _python_deps():
     )
     pip_install(
         name = "kafka_pip3",
-        requirements = "@envoy//source/extensions/filters/network/kafka:requirements.txt",
```

Comment:
not immediately viable for _this_ PR, but `repositories_extra.bzl` but AFAICT this file is going to be included in https://github.com/envoyproxy/envoy/blob/main/WORKSPACE#L15 in every build

in future we might want to separate extra repositories and extra things from `bazel/foreign_cc`, as we wouldn't want them in the main binary build

that would allow contributors to add their own static dependencies etc.

Reply:
Yeah I agree we can further improve. AFAIK though if kafka is never built bazel should not do anything, right?

Reply:
ah right! it's never going to be referenced